src/libtiled/mapreader.cpp

```diff
@@ -456,13 +465,6 @@ void MapReaderPrivate::readTilesetImage(Tileset *tileset)
 
     const QXmlStreamAttributes atts = xml.attributes();
     QString source = atts.value(QLatin1String("source")).toString();
-    QString trans = atts.value(QLatin1String("trans")).toString();
-
-    if (!trans.isEmpty()) {
-        if (!trans.startsWith(QLatin1Char('#')))
-            trans.prepend(QLatin1Char('#'));
-        tileset->setTransparentColor(QColor(trans));
-    }
```

Comment:
Hmm, you can't just remove this code, since that breaks backwards compatibility. In general I'm not sure whether this change is really worth breaking compatibility at all. :-/